resources/assets/less/bem/beatmap-discussion-new.less

```diff
@@ -49,6 +46,23 @@
     }
   }
 
+  &__footer-column {
```

Comment:
the commit message says less confusing name but there are two footer columns and this only for one specific column, not both 🤨